src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Extensions/AssertExtensionsTests.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections.Generic;
+using Xunit;
+using Xunit.Sdk;
+using static GovUk.Education.ExploreEducationStatistics.Common.Services.CollectionUtils;
+using static GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions.AssertExtensions;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions;
+
+public class AssertExtensionsTests
+{
+    private record Company(string Name, List<Person> Employees);
+
+    private record Person(string Name, int Age);
+
+    [Fact]
+    public void AssertDeepEqualsTo_Simple()
```

Comment:
Could group methods into nested classes, then we don't need to prefix each method with the method name in a repetitive way e.g.

```cs
public class AssertDeepEqualsToTests
{
  [Fact]
  public void Simple() { ... }
  
  [Fact]
  public void Simple_NotEqual() { ... } 

  // etc...
}
```

Reply:
Good idea - done!